io_scene_godot/converters/simple_nodes.py

```diff
@@ -165,3 +168,68 @@ def export_light_node(escn_file, export_settings, node, parent_gd_node):
                           light_node, node.data, 'light')
 
     return light_node
+
+
+def export_curve_node(escn_file, export_settings, node, parent_gd_node):
+    """Export a curve to a Path node, with a child mesh."""
+    splines = node.data.splines
+    if len(splines) == 0:
+        logging.warning("Curve %s has no splines", node.name)
+        return None
+
+    spline = splines[0]
+
+    if spline.type != "BEZIER":
+        return export_mesh_node(
+            escn_file, export_settings, node, parent_gd_node
+        )
+
+    if len(splines) > 1:
+        logging.warning(
+            "Bezier %s has >1 splines, only first will be exported", node.name
+        )
+
+    points = Array("PoolVector3Array(")
+    tilts = Array("PoolRealArray(")
+    axis_correct = mathutils.Matrix((
+        (1, 0, 0),  # X in blender is  X in Godot
+        (0, 0, -1),  # Y in blender is -Z in Godot
+        (0, 1, 0),  # Z in blender is  Y in Godot
+    )).normalized()
+
+    src_points = spline.bezier_points
+    if spline.use_cyclic_u:
```

Comment:
To be robust, might be good to add a check to ensure `src_points` is not empty.

Reply:
splines with no points are not included in `splines` at all. That is, if you have 1 spline with no points, `splines` actually has `len == 0`.

Reply:
Got it. Thanks for checking.